### DISCUSSION TITLE
Translation of SpotBugs manual

### DISCUSSION THREAD

**Discussion by luana-martins** [spotbugs/spotbugs#1786] [Q&A] [answered]
Hi! 

I am new to contributing to Spotbugs, and I would like to translate the SpotBugs manual to Brazilian Portuguese (PT-BR). 
Do you think it is a good contribution for SportBugs? 

If yes, could you (@KengoTODA) help me configure the ReadTheDocs for the new language? 

Thanks! :smiley:

**Comment by KengoTODA** [comment, marked as answer]
Thanks for your contribution, @luana-martins ! 👍    
I've created a ReadTheDocs project [spotbugs-pt-br](https://readthedocs.org/dashboard/spotbugs-pt-br), and added it as a [translation for spotbugs](https://readthedocs.org/dashboard/spotbugs/translations/). I'll fix when I found more TODOs to support a new translation.

I have a request: when you submit a PR, please update the following part in `docs/build.sh` to build `.po` files even for Brazilian Portuguese:

https://github.com/spotbugs/spotbugs/blob/636799d144343d090c9bb0f5a31249c9a3a2cc45/docs/build.sh#L9-L10

It helps us to keep `.po` files updated. Thanks in advance!

## Note: Settings of the spotbugs-pt-br project

![image](https://user-images.githubusercontent.com/505751/139773813-187409df-449c-489d-9bb1-c8943a773c9d.png)

## Note: Translation configuration of the spotbugs project

![image](https://user-images.githubusercontent.com/505751/139773790-9d728865-6c6e-478e-9e4c-dbdbef30c933.png)
